playbooks/prepare_host.yaml

```diff
@@ -35,7 +35,7 @@
       - name: Install virt module
         shell: |
           dnf module disable -y virt:rhel
-          dnf module enable -y virt:"{{ rhsm_release }}"
+          dnf module enable -y virt:{{ rhsm_release }}
```

Comment:
I guess it doesn't matter as long as rhsm_release is a single word, which it always should be.

I wouldn't want to rely on the lack of escaping here to allow passing other arguments, for eg.